src/python_testing/TC_IDM_4_2.py

```diff
@@ -67,6 +67,29 @@ async def get_idle_mode_duration_sec(self, ctrl, ep=ROOT_NODE_ENDPOINT_ID):
             attribute=Clusters.IcdManagement.Attributes.IdleModeDuration
         )
 
+    def get_attribute_value_wait(self, sub, typed_attr_path):
+        start_time = time.time()
+        timeout = 10
+        increment = 0.1
```

Comment:
I think we may have a few of these sprinkled in the code. How about adding a retry library in the testing requirements like https://tenacity.readthedocs.io/en/latest/ or maybe there are others, so we can read code like `@retry(stop=stop_after_delay(10), wait=wait_fixed(0.1), retry=retry_if_result(is_empty)))` or something like that.

Reply:
ugh...I hope we don't. If we do, that seems like a test red flag.

Reply:
I put up a PR to disable this test in the CI. The test plan has been updated so we don't have to do this. I'm hoping we can just skip this retry at this point and fix the test properly. This was meant to be a quick fix-forward to get this to stop flaking.

https://github.com/project-chip/connectedhomeip/pull/32378